man/systemd-udevd.service.xml

```diff
@@ -105,6 +105,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-s=</option></term>
+        <term><option>--timeout-signal=</option></term>
+        <listitem>
+          <para>Set the signal which systemd-udevd will send to the forked off
```

Comment:
maybe say "UNIX process signal"

Reply:
enclose `systemd-udevd` in `<filename>` or so

Reply:
"UNIX process signal" sounds really weird TBH and also it isn't used very often I can see only handful of references.

Reply:
Marking this as resolved since I've fixed `<filename>` part.